catkin_tools/verbs/catkin_build/job.py

```diff
@@ -139,6 +141,32 @@ def __init__(self, package, package_path, context, force_cmake):
         Job.__init__(self, package, package_path, context, force_cmake)
         self.commands = self.get_commands()
 
+    def get_multiarch(self):
+        if not sys.platform.lower().startswith('linux'):
+            return ''
+        # this function returns the suffix for lib directories on supported systems or an empty string
+        # it uses two step approach to look for multiarch: first run gcc -print-multiarch and if
+        # failed try to run dpkg-architecture
+        throws = False
+        try:
+            p = subprocess.Popen(
+                ['gcc', '-print-multiarch'],
+                stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            out, err = p.communicate()
+        except:
```

Comment:
At first I was gonna let this pass, but it is bothering me...

It's not good practice to catch and silently pass on all exceptions. I know it's more work, but it would be better if you caught specific errors, something like this: `except (OSError, FileNotFoundError):`